RetrospectiveExtension.Frontend/dal/boardDataService.tsx

```diff
@@ -27,6 +36,7 @@ class BoardDataService {
       modifiedDate: new Date(Date.now()),
       shouldShowFeedbackAfterCollect: shouldShowFeedbackAfterCollect ? shouldShowFeedbackAfterCollect : false,
       displayPrimeDirective: displayPrimeDirective ? displayPrimeDirective : false,
+      allowCrossColumnGroups: allowCrossColumnGroups ? allowCrossColumnGroups : false,
```

Comment:
Could this work instead? 

```
allowCrossColumnGroups: allowCrossColumnGroups ?? false
```

Reply:
And will apply this pattern to the others as well! Thank you for catching these.